### PR TITLE
FHIR data types and field groups

### PR DESCRIPTION
**Pull request by amadobe**
[PLAT-197603](https://jira.corp.adobe.com/browse/PLAT-197603)
[PLAT-197605](https://jira.corp.adobe.com/browse/PLAT-197605)
[PLAT-197746](https://jira.corp.adobe.com/browse/PLAT-197746)

### CODE REVIEW

File: extensions/industry/healthcare/fhir/fieldgroups/careplan.schema.json [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+   "title":"Care Plan",
+   "type":"object",
+   "meta:tags":{
+      "industry":[
+         "healthcare_services"
+      ]
+   },
+   "meta:intendedToExtend":[
+      "https://ns.adobe.com/xdm/context/profile",
+      "https://ns.adobe.com/xdm/context/account"
```
Comment (anandphatak) [inline extensions/industry/healthcare/fhir/fieldgroups/careplan.schema.json:19]: Is there a reson this is required in the b2b classes? `https://ns.adobe.com/xdm/context/account` is a b2b class

File: extensions/industry/healthcare/fhir/fieldgroups/goal.schema.json [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+   "$schema":"http://json-schema.org/draft-06/schema#",
+   "title":"Goal",
+   "type":"object",
+   "meta:tags":{
+      "industry":[
+         "healthcare_services"
+      ]
+   },
+   "meta:intendedToExtend":[
+      "https://ns.adobe.com/xdm/context/account",
```
Comment (anandphatak) [inline extensions/industry/healthcare/fhir/fieldgroups/goal.schema.json:18]: Same question as above for b2b class

File: extensions/industry/healthcare/fhir/fieldgroups/coverage.schema.json [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+               "$ref":"https://ns.fhir.org/datatypes/reference"
+            },
+            "fhir:class":{
+               "title":"Class",
+               "description":"A suite of underwriter specific classifiers.",
+               "type":"array",
+               "items":{
+                  "type":"object",
+                  "properties":{
+                     "xdm:ype":{
```
Comment (anandphatak) [inline extensions/industry/healthcare/fhir/fieldgroups/coverage.schema.json:141]: Pl. change to `xdm:type`

**Comment by amadobe** [comment]
@anandphatak since there were issues from rebasing this branch. I will create a new branch and PR for these changes.

**Pull request closed without merging**